demisto_sdk/commands/unify_packs/tests/packs_unifier_test.py

```diff
@@ -0,0 +1,153 @@
+from contextlib import contextmanager
+from pathlib import Path
+from shutil import rmtree, unpack_archive
+
+import click
+import demisto_client
+import pytest
+from demisto_client.demisto_api import DefaultApi
+from demisto_sdk.__main__ import unify_packs
+from demisto_sdk.commands.common.tools import src_root
+from demisto_sdk.commands.upload import uploader
+from demisto_sdk.commands.upload.uploader import Uploader
+from packaging.version import parse
+
+UNIT_TEST_DATA = (src_root() / 'commands' / 'unify_packs' / 'tests' / 'data')
+TEST_DATA = src_root() / 'tests' / 'test_files'
+TEST_CONTENT_REPO = TEST_DATA / 'content_slim'
+TEST_PACK_DIR = src_root() / 'Packs'
+TEST_PACK_PATH = src_root().parent / 'Packs' / 'TestPack'
```

Comment:
Did you check that none of those constants exists in the `constants.py`/`test_constants.py`?